servers/src/common/types.rs

```diff
@@ -163,19 +163,16 @@ impl ServerConfig {
 		// check [server.p2p_config.capabilities] with 'archive_mode' in [server]
 		if let Some(archive) = self.archive_mode {
 			// note: slog not available before config loaded, only print here.
-			if archive != self
-				.p2p_config
-				.capabilities
-				.contains(p2p::Capabilities::FULL_HIST)
+			if archive
+				!= self
```

Comment:
What's your `rustfmt` version?  the format is a little bit strange. 
Or what's your rust version?